recharges/tests.py

```diff
@@ -376,3 +456,17 @@ def test_hotsocket_get_airtime_unrecoverable(self):
         # Check
         self.assertEqual(result.get(),
                          "airtime request for +277244555 is unrecoverable")
+
+    def test_Check_Hotsocket_Status(self):
+        recharge_id = self.make_recharge()
+        result = hotsocket_status(recharge_id)
+        self.assertEqual(result, "recharge is succesful")
+
+
+class TestRechrgeStatus(TaskTestCase):
+    def test_prep_hotsocket_status_dict(self):
+        self.make_account()
+        result = hotsocket_status.prep_hotsocket_status_dict(recharge_id=1003,
+                                                             hs_reference=4507)
+        self.assertEqual(result['reference'], 11003)
+        self.assertEqual(result['hotsocket_ref'], 4507)
```

Comment:
You should also be checking for data that you haven't passed through, like the `token` and `username`.